README.md

```diff
@@ -1,26 +1,51 @@
-## Project board
-https://app.zenhub.com/workspaces/commit-zero-5da8decc7046a60001c6db44/board?filterLogic=any&repos=203630543,247773730,257676371,258369081
+# Overview
+A set of templates meant to work with [Zero][zero], the templated result is a ready to scale infrastructure boilerplate built on top of AWS EKS baked with all best practices we have accumulated.
```

Comment:
fyi: just realized that MD is case agnostic with in-line links so even if you define the hyperlink as [zero] referencing it as [Zero] still works 😅
```suggestion
A set of templates meant to work with [Zero], the templated result is a ready to scale infrastructure boilerplate built on top of AWS EKS baked with all best practices we have accumulated.
```